### PR TITLE
Added implementation for WAIT command

### PR DESCRIPTION
**Pull request by natethebosch**
Reference: https://redis.io/commands/wait

### CODE REVIEW

**Comment by natethebosch** [comment]
Not sure if this command was purposely left out of the commands list... but any rate here's an implementation.

File: commands.go [hunk 1]
```diff
@@ -271,6 +271,13 @@ func (c *cmdable) Ping() *StatusCmd {
 	return cmd
 }
 
+func (c *cmdable) Wait(n_slaves uint, n_seconds uint) *IntCmd {
```
Comment (vmihailenco) [inline commands.go]: Go convention is to use camelCased var names. So it should be `numSlaves, numSeconds`. Overall I suggest to change types to `numSlaves int, timeout time.Duration` so it reads better:

```
client.Wait(1, time.Minute)
```

Comment (natethebosch) [inline commands.go]: Good suggestions, I've made those changes

**Comment by vmihailenco** [comment]
Thanks - merging.

**Pull request merged**